test_rstcheck.py

```diff
@@ -81,14 +84,11 @@ def test_check_with_extra_blank_lines(self):
 .. code-block:: python
 
     print(
-
```

Comment:
Any reason why these blank lines were removed? It kind of makes the name of the test obsolete (`test_check_with_extra_blank_lines()`).